### ISSUE TITLE
Cannot open sidebar using toggle button when VoiceOver is active in iOS

### ISSUE THREAD

**Issue by robertknight** [hypothesis/client#1765]
# Steps to reproduce
1. On iOS, activate VoiceOver. (nb. Before activating VO, I recommend changing the "Typing" mode in VO settings to "Direct touch typing", as using the on-screen keyboard to enter a URL in Safari is painfully slow otherwise)
2. Open Safari and navigate to https://hypothes.is/docs/help
3. Select and activate the sidebar toggle button to close it
4. Select and try to activate the sidebar toggle button to open it again

### Expected behaviour
It should be possible to open the sidebar by tapping on the toggle button.

### Actual behaviour
The sidebar flashes open very briefly and then immediately closes.

### Browser/system information
Tested in Safari on an iPhone X running iOS 13.3.1. I expect the same issue affects iPad OS.

**Comment by robertknight** [comment]
From a little debugging, it looks like the `onElementTouchStart` handling in `guest.coffee` is making this happen. The intent is to hide the sidebar when the user taps in an area on the page that is somewhere outside the sidebar, however these events on the toolbar trigger the same thing happening.

The problem doesn't arise with `click` events on the toolbar because the toolbar buttons stop propagation of those events up to the document.
